pkg/issuer/acme/setup.go

```diff
@@ -195,6 +218,7 @@ func (a *Acme) Setup(ctx context.Context) error {
 	log.Info("verified existing registration with ACME server")
 	apiutil.SetIssuerCondition(a.issuer, v1alpha1.IssuerConditionReady, v1alpha1.ConditionTrue, successAccountRegistered, messageAccountRegistered)
 	a.issuer.GetStatus().ACMEStatus().URI = account.URL
+	a.issuer.GetStatus().ACMEStatus().LastRegisteredEmail = registeredEmail
```

Comment:
👍 great, this means we will only ever set `LastRegisteredEmail` to the value received from the ACME server.